### PR TITLE
`Popup`: Adds TextPopupElement

### PR DESCRIPTION
**Pull request by eri9000**
Adds TextPopupElement composable.
This composable can render HTML text.

### CODE REVIEW

File: toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/ExpandableCard.kt [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                    )
+                    IconButton(onClick = { expanded = !expanded }) {
+                        Icon(
+                            imageVector = if (expanded) Icons.Default.KeyboardArrowUp else Icons.Default.KeyboardArrowDown,
+                            contentDescription = "More"
+                        )
+                    }
+                }
+            }
+
```
Comment (sorenoid) [inline toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/ExpandableCard.kt:60]: There should probably be a Spacer here.

File: toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/ExpandableCard.kt [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    ) {
+        Column {
+            if (title.isNotEmpty()) {
+                Row(modifier = Modifier.fillMaxWidth(),horizontalArrangement = Arrangement.SpaceBetween) {
+                    Text(
+                        title,
+                        modifier = Modifier.padding(8.dp),
+                        style = MaterialTheme.typography.titleMedium,
+                    )
+                    IconButton(onClick = { expanded = !expanded }) {
```
Comment (sorenoid) [inline toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/ExpandableCard.kt:52]: Is IconButton necessary? It seems like an Icon wrapped in a CrossFade animation will work. I think at any rate the animation is necessary.

File: toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/ExpandableCard.kt [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                        Icon(
+                            imageVector = if (expanded) Icons.Default.KeyboardArrowUp else Icons.Default.KeyboardArrowDown,
```
Comment (sorenoid) [inline toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/ExpandableCard.kt:54]: I think the carets are not centered in their bitmaps. Better to use `Icons.Rounded.ExpandLess` and `Icons.Rounded.ExpandMore`

**Review by sorenoid — CHANGES_REQUESTED** [review]
Looks nice. Some small suggestions.

File: toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/ExpandableCard.kt (hunk 1 above)
Comment (eri9000) [reply toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/ExpandableCard.kt:60]: I've added a spaced only when there's a title.

**Review by sorenoid — APPROVED** [review]

**Pull request merged**